vcd/sample_vcd_test_config.json

```diff
@@ -11,7 +11,14 @@
     "allowInsecure": true,
     "//": "tfAcceptanceTests is needed to allow the whole test suite to run",
     "//": "Warning: it may cost time (from 14 to 40 min) and possibly money",
-    "tfAcceptanceTests": false
+    "tfAcceptanceTests": false,
+    "//": "If enabled, the test suite will cache the provider connection for up to 20 minutes",
+    "//": "and then renew it automatically. This will save some heavy network traffic.",
+    "//": "The default is false",
```

Comment:
You mean default if UseVcdConnectionCache is not set in the config file at all? It's a bit strange to read "default is false" and then see ""UseVcdConnectionCache": true" a line below. Better rephrase the comment to something like "Disabled by default if UseVcdConnectionCache is not set".

Reply:
done